lua/telescope/make_entry.lua

```diff
@@ -918,11 +918,11 @@ function make_entry.gen_from_ctags(opts)
   }
   local hidden = utils.is_path_hidden(opts)
   if not hidden then
-    table.insert(display_items, 1, { width = 30 })
+    table.insert(display_items, 1, { width = opts.fname_width or 30 })
   end
 
   if opts.show_line then
-    table.insert(display_items, 1, { width = 30 })
+    table.insert(display_items, 1, { width = opts.fname_width or 30 })
```

Comment:
Not here. (or make another opts) because this is not about the contents of the line not the filename